# Hope Street / Highland Ave Properties

In [23]:
from geopandas import GeoDataFrame

In [24]:
df = df[df.OWNER1.isin([
    "BAILEY JOHN G",
    "WITHINGTON DENNET W TRUSTEE SUN TRUST"
])][["TOTAL_VAL", "LOT_SIZE", "SITE_ADDR", "OWNER1", "LS_BOOK", "LS_PAGE", "geom"]]

## Table

There are five properties that John Bailey, or the sun trust, own which we are talking to him about buying together:

In [25]:
df

TOTAL_VAL  LOT_SIZE            SITE_ADDR  \
5660       5100      0.84              HOPE ST   
5661      60900      3.50              HOPE ST   
5679      65500      6.24              HOPE ST   
5680       6700      1.00      HIGHLAND AVE EX   
8275      89800      8.10  172 HIGHLAND AVE EX   

                                     OWNER1 LS_BOOK LS_PAGE  \
5660  WITHINGTON DENNET W TRUSTEE SUN TRUST    5704      87   
5661  WITHINGTON DENNET W TRUSTEE SUN TRUST    5704      87   
5679  WITHINGTON DENNET W TRUSTEE SUN TRUST    5704      87   
5680  WITHINGTON DENNET W TRUSTEE SUN TRUST    5704      87   
8275                          BAILEY JOHN G    5169     130   

                                                   geom  
5660  POLYGON ((110386.944 925398.655, 110302.269 92...  
5661  POLYGON ((110418.040 925309.733, 110385.393 92...  
5679  POLYGON ((110330.406 925522.064, 110374.077 92...  
5680  POLYGON ((110302.025 925777.273, 110301.942 92...  
8275  POLYGON ((110316.856 925756.729, 110303.615 92...

## Map

We can see in the map that these lots surround the existing buildings at 172 Highland, which were split off by John previously and sold for $500k.

In [26]:
df.explore()

## Total Acreage

In [9]:
df.LOT_SIZE.sum()

19.68

## Total Value

In [23]:
print('${:,.0f}'.format(df.TOTAL_VAL.sum()))

$228,000
